### ISSUE TITLE
InfoWindow on mouseover should be disabled. At least for touch devices.

### ISSUE THREAD

**Issue by rnarian** [AHAAAAAAA/PokemonGo-Map#1525]
Right now InfoWindows open on mouseover. This was introduced with #544. Unfortunately this makes the map almost unusable on mobile. When zooming or dragging the map countless InfoWindows will open/close unexpectedly and the whole screen starts flickering.

![juli-23-2016 15-46-49](https://cloud.githubusercontent.com/assets/1174951/17078182/b809cd00-50ec-11e6-86c8-92b47332ffef.gif)

Since the map will most likely be used on a mobile device while hunting Pokémon—not on a desktop pc—this feature is unnecessary in most use cases.

Also when looking at a regular Google Maps map InfoWindows won't open on mouseover either, so this doesn't even seem right on desktop pcs, even though its kind of working.

**Comment by frdmn** [comment]
:+1:

**Comment by neobithia** [comment]
Disable it on mobile but don't get rid of it.  I use the map on a laptop while hunting.  My laptop is using wifi from my phone.

**Comment by invisiblek** [comment]
https://github.com/AHAAAAAAA/PokemonGo-Map/pull/1679

**Comment by frdmn** [comment]
@invisiblek Awesome!

**Comment by rnarian** [comment]
@invisiblek Thank you! :+1:

**Comment by trickpattyFH20** [comment]
I am very against the idea behind PR #1679 . It completely disables the infoWindow popup for mobile devices.

Seeing the despawn countdown timer is one of the most important features of this application.

The main problem on mobile devices is that the screen jumps (autoPan) when you open an infoWindow so that the entire popup fits on screen.

I created a pull request #1811 to prevent the screen from jumping around when a popup is opened.

I really hope that #1679 does not get pulled, it would be detrimental to the mobile experience to completely disable the popups. @frdmn

**Comment by invisiblek** [comment]
@trickpattyFH20 https://github.com/AHAAAAAAA/PokemonGo-Map/pull/1679 has now been updated. It disables the popups on mouseover **during** touch. Simply tapping the pokemon still brings up the info window (it always has through that patch)

**Comment by trickpattyFH20** [comment]
@invisiblek nice, had me worried there :)

**Comment by MonkeyLeeT** [comment]
#1811 Seems to work better in a simpler way. Can you guys test it? (It's working for me and map is not paning around).

**Comment by rnarian** [comment]
No, this isn't fixed unfortunately. It's different, but it's still there. When a zoom or a drag event starts with a touch on a marker, the page isn't zoomed or dragged but a body scroll is triggered. Now I have to try to tap areas without markers to interact with the map properly. When many markers are displayed this is almost impossible because the map is laced with markers.

Again. All this was introduced by triggering the marker on mouseover. Without this change the map would work as expected on mobile as well as on desktop. And it would work like one would expect from interacting with regular Google Maps. InfoWindows open by clicking a marker and get closed by clicking the close button. No mouseover action at all. 

This change was bad decision usability-wise (on desktop) and broke the mobile experience so I don't see why this got merged in the first place and why we don't simply remove it instead of trying to fix a problem that seems more like a hack than a solution to a problem.

**Comment by trickpattyFH20** [comment]
That's weird, I am not able to reproduce the bug on mobile devices (iPhone 6 iOS 9.3) after the merge of #1811. however I could see it potentially being a problem still for other devices. Maybe it's worth adding conditions to prevent mousover as well.

What mobile device and browser version are you using? And what build of the project are you on?

**Comment by rnarian** [comment]
iPhone 6s, iOS 9.3.3 here. Take a look at this:

|Working|Not working|
|---|---|
|![juli-26-2016 16-04-00](https://cloud.githubusercontent.com/assets/1174951/17141010/d46a47b2-534a-11e6-9200-d07d2bfc7af2.gif)|![juli-26-2016 16-04-50](https://cloud.githubusercontent.com/assets/1174951/17141025/e4be7c00-534a-11e6-8577-c6ddd436393b.gif)|

> When a zoom or a drag event starts with a touch on a marker, the page isn't zoomed or dragged but a body scroll is triggered. Now I have to try to tap areas without markers to interact with the map properly. When many markers are displayed this is almost impossible because the map is laced with markers.

**Comment by trickpattyFH20** [comment]
gothcha. problem is clear to me now (although in a technical sense this a separate issue than the original one solved by #1811) @frdmn @invisiblek can anyone test if removing the mouseover listeners fixes this and make a PR?
